src/pages/customers/index.js

```diff
@@ -80,6 +89,12 @@ refreshTable = () => {
                 })
               }
             })}
+            SubComponent={row => {
```

Comment:
We don't use TableDropdown anymore do we? we hijacked the onClick function so this is no longer in use. You can delete is and probably delete the old files for table dropdown as well since we don't use them anymore

Reply:
In order to have an edit button to open the modal, we need to have a subtable to the main table. You're right, this doesn't need to be TableDropdown, but it does need to be a ReactTable, so I replaced this line with an empty ReactTable instead. As for deleting the old TableDropdown files, I'd like to keep them around to prepare for orals and then remove them afterwards.